template/golang-http/go.mod

```diff
@@ -2,4 +2,9 @@ module handler
 
 go 1.18
 
-require github.com/openfaas/templates-sdk/go-http v0.0.0-20220408082716-5981c545cb03
```

Comment:
this part is copied from this repository times

https://github.com/openfaas/golang-http-template/tree/eb793ff6f2749e1ebb02f4c235a78d7602102d7e